python/ray/actor.py

```diff
@@ -599,11 +568,20 @@ def method(self):
             lifetime=lifetime,
             placement_group=placement_group,
             placement_group_bundle_index=placement_group_bundle_index,
-            placement_group_capture_child_tasks=(placement_group_capture_child_tasks),
-            runtime_env=new_runtime_env,
+            placement_group_capture_child_tasks=placement_group_capture_child_tasks,
+            runtime_env=runtime_env,
             max_pending_calls=max_pending_calls,
             scheduling_strategy=scheduling_strategy,
         )
+        # get user input options and override original options
+        cls_options = options.user_input_filter(cls_options)
+        cls_options = {**self._option_dict, **cls_options}
+        options.validate_actor_options(cls_options, include_options_only=True)
+        if "runtime_env" in cls_options:
+            cls_options["runtime_env"] = parse_runtime_env(cls_options["runtime_env"])
+        # TODO(suquark): This is a temporary fix, because actor._remote does not support
+        # "concurrency_groups".
+        cls_options.pop("concurrency_groups", None)
```

Comment:
This is likely inconsistency in our current API. Will address it in later PRs.

Reply:
Yes. It only supports `.remote` according to the doc.